src/index.js

```diff
@@ -149,23 +113,17 @@ class CookieConsent extends Component {
     const {
       cookieName,
       declineCookieValue,
-      expires,
```

Comment:
Were these unused?

Reply:
Nowhere, I think. That's why I remove it :)

Reply:
I'm more worried why they weren't being used tbh, I'll look into them later :smile: